### PR TITLE
Add a mac M1 compatible configuration

### PR DESCRIPTION
**Pull request by EnoF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,30 +188,42 @@ in devnet.yaml that store the database on a named value or on the host.
...
-        
-        You can also use the commands listed elsewhere in this document to query into the node container directly if you reference the right container name: devnet-bootstrap-node-1
+
+- You can delete this folder to reset your devnet blockchain, or back it up to make a savestate
+  - _Turn off the node container before doing stuff to this folder_
+  - The structure must match the above with a 0/ folder
+  - You do not _need_ the sqlite folder if you want to save backup space, as chainweb-node will rebuild one from the rocksDb if present
+- If you are starting from a fresh database, you may wish to wait for the block height of each chain to pass the latest feature fork for devnet
+
+  - As of 2.15, this is a height of 165. It takes an hour or so to reach this from scratch.
```
Comment (edmundnoble) [inline README.md]:
```suggestion
  - As of 2.18, this is a height of 500. It takes four hours or so to reach this from scratch.
```

Comment (edmundnoble) [reply README.md]: These docs can be improved in other ways, but this is at least something.

**Review by edmundnoble — COMMENTED** [review]
I wonder if we can just add the `platform` spec to the docker-compose.yaml and docker-compose.minimal.yaml files to avoid proliferating more configs.

**Comment by EnoF** [comment]
That would be great, I just don't know how. If you do, I'm happy to change the config to match that.

**Comment by edmundnoble** [comment]
We can just add this for now if it helps Eileen.

**Pull request merged**